app/config/ConfigDecorator.scala

```diff
@@ -123,6 +124,15 @@ class ConfigDecorator @Inject()(
   lazy val makeAPaymentUrl = s"$payApiUrl/pay-api/pta/sa/journey/start"
   lazy val deskproToken = "PTA"
 
+  lazy val accessibilityStatementToggle: Boolean =
+    runModeConfiguration.getOptional[Boolean](s"accessibility-statement.toggle").getOrElse(false)
+  lazy val accessibilityBaseUrl = runModeConfiguration.underlying.getString(s"accessibility-statement.baseUrl")
```

Comment:
Can we please use getString("accessibility-statement.baseUrl") instead of runModeConfiguration.underlying.getString(s"accessibility-statement.baseUrl") .